### PR TITLE
golo 3.4.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by samford — DISMISSED** [review]

File: Formula/golo.rb [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
+  sha256 "867c462a41a20e4b7dc1aef461b809d193a505c2a757477b147f0e30235bd545"
   license "EPL-2.0"
-  revision 2
   head "https://github.com/eclipse/golo-lang.git"
 
   livecheck do
-    url "https://golo-lang.org/download/"
-    regex(/href=.*?golo[._-]v?(\d+(?:\.\d+)+)\.zip/i)
+    url "https://github.com/eclipse/golo-lang.git"
+    regex(%r{^release/(\d+(?:\.\d+)+)$}i)
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/golo.rb:11]: got it!

**Comment by chenrui333** [comment]
Should good for the merge after rerun. (retested in my local)

**Comment by cho-m** [comment]
Doesn't look like CI ran correctly as it only shows Triage output.

**Review by Moisan — APPROVED** [review]

**Review by samford — COMMENTED** [review]
The `livecheck` block looks good to me at this point 👍

**Pull request closed without merging**